HelpSource/Classes/Dictionary.schelp

```diff
@@ -99,21 +103,21 @@ Access the value associated with the key. If the key does not exist, return the
 method::keys
 Return a link::Classes/Set:: of all keys.
 code::
-d = (hello: 9, whello: "world");
+d = Dictionary.newFrom(List[\hello, 9, \whello, "world"]);
```

Comment:
Is there a reason to use `List` explicitly here? (similarly in the following examples)

Reply:
I only used "List" because, that is the form in which the method::newFrom is explained in the beginning of the document and it might be easier to understand, as it is more explicit than the short version 
`Dictionary[\hello, 9, \whello, "world"]`
In order to get the examples more consistent I could use only one version (short or long) throughout the document and list all possible initializations in the top under the classmethods section.

Reply:
Oh, got it. I agree consistency would be best.
I was actually thinking about just dropping `List`, so `d = Dictionary.newFrom([\hello, 9, \whello, "world"]);` What do you think? In that case I'd propose to use that form at the explanation of `-newFrom`

Reply:
@dyfer: Ok, I will stick to `d = Dictionary.newFrom([\hello, 9, \whello, "world"]); `, only where `newFrom` is used in an example. 
Or do you think all examples in the whole document should use the same form of initialization?

Reply:
I think only where it was `newFrom` would be fine.